docs/snippets/common/button-story-primary-long-name.js.mdx

```diff
@@ -1,10 +1,16 @@
 ```js
 // Button.stories.js | Button.stories.jsx | Button.stories.ts | Button.stories.tsx
 
-export const PrimaryLongName = Template.bind({});
+import Button from './Button';
 
-PrimaryLongName.args = {
-  ...Primary.args,
-  label: 'Primary with a really long name',
-}
-```
\ No newline at end of file
+export default {
+  component: Button,
+};
+
+export const PrimaryLongName = {
+  args: {
+    ...Primary.args,
```

Comment:
Where are Primary's args defined?

Reply:
@darleendenno this example is following up the example above, for context see [here](https://storybook.js.org/docs/react/writing-stories/args#story-args). The first story is named "Primary" and this follows up on it.

Reply:
That makes sense. Thank you!